addons/account/models/account.py

```diff
@@ -702,6 +702,18 @@ def copy(self, default=None):
         default = dict(default or {}, name=_("%s (Copy)") % self.name)
         return super(AccountTax, self).copy(default=default)
 
+    @api.multi
+    def name_get(self):
+        if not self._context.get('include_taxes_type'):
+            return super(AccountTax, self).name_get()
+        result = []
+        type_tax_use_selection = dict(self._fields['type_tax_use'].selection)
```

Comment:
IIRC `.selection` is not translated (ping @rco-odoo do you confirm?) so that probably isn't the one you want...